cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -1516,6 +1527,12 @@ def _add_compute_resource_launch_template(
                                 "EFSSharedDirs": to_comma_separated_string(
                                     self._shared_storage_mount_dirs[SharedStorageType.EFS]
                                 ),
+                                "EFSEncryptionInTransits": to_comma_separated_string(
+                                    self._shared_storage_attributes[SharedStorageType.EFS]["EncryptionInTransits"]
+                                ),
+                                "EFSIamAuthorizations": to_comma_separated_string(
+                                    self._shared_storage_attributes[SharedStorageType.EFS]["IamAuthorizations"]
+                                ),
```

Comment:
You also need to change https://github.com/aws/aws-parallelcluster/blob/develop/cli/src/pcluster/resources/compute_node/user_data.sh#L67

You should add the following lines in the file:
```
"efs_encryption_in_transits": "${EFSEncryptionInTransits}",
"efs_iam_authorizations": "${EFSIamAuthorizations}",
```